### ISSUE TITLE
System.Reflection.ReflectionTypeLoadException when using generated protocol buffer classes

### ISSUE THREAD

**Issue by daulet** [Azure/azure-webjobs-sdk#1681]
Function throws the following (at least when running locally):

_Warning: Only got partial types from assembly: ...., Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
Exception message: System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at Microsoft.Azure.WebJobs.Host.Indexers.DefaultTypeLocator.FindTypes(Assembly assembly, IEnumerable`1 extensionAssemblies) in C:\projects\azure-webjobs-sdk-rqm4t\src\Microsoft.Azure.WebJobs.Host\Indexers\DefaultTypeLocator.cs:line 120_

#### Repro steps

1. Create HTTP trigger function (might repro for any trigger), include [this file](https://gist.github.com/daulet/ea7c50a6f06e2bd45ed6b2f14c9603a5). It is a generated file based on simple proto file:

```
syntax = "proto3";
package proof;

message SampleType {
    int32 sample_field = 1;
}
```

You will also need to reference Google.Protobuf package.

2. Run it locally

#### Related information 

* Microsoft.NET.Sdk.Functions (1.0.13)
* Google.Protobuf (3.5.1)
* AzureFunctionsVersion v2

**Comment by naveenduddi** [comment]
Facing this issue. Any work around or solution?

**Comment by fabiocav** [comment]
This has been addressed by the fix put in place for #1722 
All problem details will be displayed, allowing the root cause to be identified.
